src/cursor/cur_std.c

```diff
@@ -1113,6 +1113,55 @@ __wt_cursor_reconfigure(WT_CURSOR *cursor, const char *config)
     API_END_RET(session, ret);
 }
 
+/*
+ * __wt_cursor_largest_key --
+ *     WT_CURSOR->largest_key default implementation..
+ */
+int
+__wt_cursor_largest_key(WT_CURSOR *cursor)
+{
+    WT_DECL_ITEM(key);
+    WT_DECL_RET;
+    WT_SESSION_IMPL *session;
+    WT_TXN *txn;
+    bool ignore_tombstone;
+
+    txn = NULL;
```

Comment:
This line is going to get flagged by a static analyzer as assigned but never read; we don't really need a local variable?